src/main/java/li/cil/sedna/instruction/decoder/DecoderGenerator.java

```diff
@@ -219,7 +219,7 @@ protected final class GeneratorContext implements Opcodes {
         public static final int LOCAL_INST = 2;
         public static final int LOCAL_PC = 3;
         public static final int LOCAL_INST_OFFSET = 5;
-        public static final int LOCAL_FIRST_FIELD = 7;
+        public static final int LOCAL_FIRST_FIELD = 8;
```

Comment:
Was this for testing or what is this change for?

Reply:
I added an argument to interpretTrace, which makes the local variables start later I think? I don't know much about asm or the JVM so that may be wrong

Reply:
Ahh, yes! Yes, you're absolutely right, thanks for reminding me!